### ISSUE TITLE
Error feedback for area selection

### ISSUE THREAD

**Issue by 5igm** [Arckenver/Nations#14]
When I selected the first point under creative mode, there are some error happening. But if I am under survive mode, there is no problem. I think it is bause player will destroy the block under creative mode.
In addition, there are still some options not in message file.
![1](https://cloud.githubusercontent.com/assets/11610914/17523712/2ab6fd4c-5e8e-11e6-978c-3a29a2adfeee.png)
Explosion
Fire
Pvp
Mobs

**Comment by Arckenver** [comment]
What exactly is the problem ? Does the light blue message appear ? What does it say ?

Also, can you specify the SpongeVanilla/SpongeForge version you are using ? Just copy paste the name of the file (e.g. `spongevanilla-1.8.9-4.2.0-BETA-348`)

For the remaining untranslatable options, those are quite hard to support, because they're shown dynamically... I'll try my best to solve this.

**Comment by 5igm** [comment]
forge-1.8.9-11.15.1.1902-1.8.9-universal
spongeforge-1.8.9-1890-4.2.0-BETA-1653
Nations-0.10.jar
@Arckenver 
Sorry, after the appearence of new error, Nation didn't work. I cannot show the error again now.

**Comment by 5igm** [comment]
@Arckenver 

```
[13:41:10] [Server thread/ERROR] [Sponge/]: Could not pass InteractBlockEvent$Primary$Impl to org.spongepowered.mod.plugin.SpongeModPluginContainer@2ef9c46d
java.lang.NullPointerException
    at com.arckenver.nations.listener.GoldenAxeListener.onPlayerLeftClick(GoldenAxeListener.java:78) ~[GoldenAxeListener.class:0.10]
    at org.spongepowered.common.event.listener.PrimaryListener_GoldenAxeListener_onPlayerLeftClick97.handle(Unknown Source) ~[?:?]
    at org.spongepowered.common.event.RegisteredListener.handle(RegisteredListener.java:95) ~[RegisteredListener.class:1.8.9-1890-4.2.0-BETA-1653]
    at org.spongepowered.mod.event.SpongeModEventManager.post(SpongeModEventManager.java:201) [SpongeModEventManager.class:1.8.9-1890-4.2.0-BETA-1653]
    at org.spongepowered.mod.event.SpongeModEventManager.post(SpongeModEventManager.java:184) [SpongeModEventManager.class:1.8.9-1890-4.2.0-BETA-1653]
    at org.spongepowered.mod.event.SpongeModEventManager.post(SpongeModEventManager.java:229) [SpongeModEventManager.class:1.8.9-1890-4.2.0-BETA-1653]
    at org.spongepowered.mod.event.SpongeModEventManager.post(SpongeModEventManager.java:214) [SpongeModEventManager.class:1.8.9-1890-4.2.0-BETA-1653]
    at org.spongepowered.common.SpongeImpl.postEvent(SpongeImpl.java:129) [SpongeImpl.class:1.8.9-1890-4.2.0-BETA-1653]
    at net.minecraft.network.NetHandlerPlayServer.redirect$handleLeftBlockClick$0(NetHandlerPlayServer.java:515) [lm.class:?]
    at net.minecraft.network.NetHandlerPlayServer.func_147345_a(NetHandlerPlayServer.java:525) [lm.class:?]
    at net.minecraft.network.play.client.C07PacketPlayerDigging.func_148833_a(SourceFile:40) [ir.class:?]
    at net.minecraft.network.play.client.C07PacketPlayerDigging.func_148833_a(SourceFile:10) [ir.class:?]
    at org.spongepowered.common.network.PacketUtil.onProcessPacket(PacketUtil.java:121) [PacketUtil.class:1.8.9-1890-4.2.0-BETA-1653]
    at net.minecraft.network.PacketThreadUtil$1.redirect$onProcessPacket$0(SourceFile:39) [fh$1.class:?]
    at net.minecraft.network.PacketThreadUtil$1.run(SourceFile:13) [fh$1.class:?]
    at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source) [?:1.8.0_101]
    at java.util.concurrent.FutureTask.run(Unknown Source) [?:1.8.0_101]
    at net.minecraft.util.Util.func_181617_a(SourceFile:44) [g.class:?]
    at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:660) [MinecraftServer.class:?]
    at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:344) [ko.class:?]
    at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:605) [MinecraftServer.class:?]
    at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:481) [MinecraftServer.class:?]
    at java.lang.Thread.run(Unknown Source) [?:1.8.0_101]
```

**Comment by Arckenver** [comment]
Ah you're right, this is a bug. Thankfully, this will be fast and easy to solve.

Thanks for reporting !

**Comment by Arckenver** [comment]
Fixed as of commit 0a7f5dbb423cf2ea4106b414b0c7b283d7ad8f14

**Comment by 5igm** [comment]
Thank you very much.
